mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/application/SocialFacadeTest.kt

```diff
@@ -109,8 +109,8 @@ class SocialFacadeTest {
     }
     val facade = SocialFacade(auth, store)
 
-    val user = facade.get("test").first()
-    Truth.assertThat(user.name).isEqualTo("test")
+    val user = facade.get("test").firstOrNull()
```

Comment:
Does this test actually pass ? I expect the `facade` not to find any document with the `uid` set to `test`, and the assertion to fail.

Reply:
It does, because I catch the error and return `null`. But I see, This is not a good approach